tools/options.py

```diff
@@ -121,3 +121,12 @@ def extract_profile(parser, options, toolchain, fallback="develop"):
                                 " supported by profile {}").format(toolchain,
                                                                    filename))
     return profile
+
+def not_disabled(parser, mcu):
```

Comment:
I would vote for is_disabled, usually logic is positive is_active, is_disabled, is_whatever. No strong preference, I care more about the name than logic being negative.

> Also, is_cortex_a_disabled implies that I'm checking if all cortex-A CPUs are disabled, not this particular CPU.

The function should have be more specific than generic name, either provide MCU parameter or something more to make it generic or change the function name specify what the function does ``is_rza1h_disabled()`` (the file for the implementation is options.py that is for anything).

these are minor things though